InnerEye/ML/run_ml.py

```diff
@@ -527,63 +521,61 @@ def register_segmentation_model(self,
         :param best_epoch_dice: Dice metric for the best epoch
         :param checkpoint_paths: Checkpoint paths to use to upload model checkpoints to AML.
         :param model_proc: whether it's a single or ensemble model.
-        :param run: If provided then the run's workspace and tags will be used to register the model.
-        :param workspace: If provided, then this workspace will be used to register the model instead of the
-        workspace associated with the provided run.
-        :param tags: If provided, then these will be used instead of the tags found in the provided run.
-        :returns AML model object, the path to the specially-deployed model if any, and a further object
-        relating to model deployment; if model_deployment_hook is None, the last two are also None.
+        :returns Tuple element 1: AML model object. Tuple element 1: The result of running the
         However if a model cannot be registered because the run is an _OfflineRun, or the model_config is not
-        for a segmentation model, None is returned instead of a model.
+        model_deployment_hook, or None if no hook was supplied.
```

Comment:
This sentence has a problem i think "or the model_config is not model_deployment_hook"